ibis-server/tests/routers/v2/connector/test_trino.py

```diff
@@ -371,3 +371,35 @@ def to_connection_info(trino: TrinoContainer):
 def to_connection_url(trino: TrinoContainer):
     info = to_connection_info(trino)
     return f"trino://{info['user']}@{info['host']}:{info['port']}/{info['catalog']}/{info['schema']}"
+
+
+def test_metadata_list_tables(self, trino: TrinoContainer):
+    connection_info = self.to_connection_info(trino)
```

Comment:
Weird, do you run the test in your local? Does this code not raise error?

Reply:
It raised the error for surely,  It does. I forget commit that 😄